_pages/covid19.html

```diff
@@ -23,13 +23,33 @@ <h2 class="demp">
     <div class="row card-row">
         <div class="medium-6 column">
             <p>
-              In order to keep conference attendees safe, DjangoCon US will have an indoor mask mandate along with a vaccine verification process in-person in San Diego as well as an online verification before the conference. If you are not familiar, we suggest you check out the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html">CDC's Domestic Travel guidelines to stay safe</a>.
+              In order to keep conference attendees safe, DjangoCon US will have an indoor mask mandate, a required proof of negative COVID test, and a vaccine verification process in-person in San Diego as well as an online verification before the conference. If you are not familiar, we suggest you check out the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html">CDC's Domestic Travel guidelines to stay safe</a>.
+              <br />
+              <br />
+              <strong>Update:</strong> On September 1, the CDC officially <a href="https://www.cdc.gov/media/releases/2022/s0901-covid-19-booster.html">recommended the new bivalent boosters</a> from Moderna and Pfizer/BioNTech. Based on these recommendations, we are now requiring one of the two conditions to be met:
+              <ul>
+                <li>
+                  If it has been fewer than 5 months since you have completed your initial vaccination schedule
+                  (2 doses for either Pfizer or Moderna or 1 dose for Johnson and Johnson or Novavax in the USA),
+                  you are not required to get an additional booster.
+                </li>
+                <li>
+                  If you have received any booster since you have completed your initial vaccination schedule,
+                   you are not required to get an additional booster at this time.
+                  </li>
+                <li>
+                  Otherwise, you are required to get a COVID booster if it is available in your country of residence.
+                </li>
+              </ol>
+              <br />
+              <br />
+              We are also requiring a negative test before coming to the registration desk on your first day of attendance. If you are taking a PCR test, it must be within 3 days of your first ticketed day (e.g. if you are attending a tutorial, the test must be taken no earlier than Thursday, October 11). If you are taking a rapid test that offers results in under 24 hours, you must take it within 24 hours of your first ticketed day.
```

Comment:
We should confirm with Share My Health that the system gives us the ability to do this.